### PR TITLE
Introduced new server lists: Any Genre 2, Choral/Barbershop (#875)

### PR DESCRIPTION
**Pull request by corrados**

### CODE REVIEW

**Comment by ann0see** [comment]
Sorry to be that guy, but why don't you use a for loop and loop through an array which contains all the lists/names? This would make adding new lists much easier.

**Comment by corrados** [comment]
What place in the code you are referring to?

> This would make adding new lists much easier.

I don't think we need more lists in the near future.

File: src/connectdlg.cpp [hunk 1]
```diff
@@ -74,10 +74,12 @@ CConnectDlg::CConnectDlg ( CClientSettings* pNSetP,
     // central server address type combo box
     cbxCentServAddrType->clear();
     cbxCentServAddrType->addItem ( csCentServAddrTypeToString ( AT_DEFAULT ) );
```
Comment (ann0see) [inline src/connectdlg.cpp:76]: You could loop through an array containing the lists starting from here. 

Could this be done with a compiler macro or something like this: https://stackoverflow.com/questions/24392000/define-a-for-loop-macro-in-c#24392091

File: src/serverdlg.cpp [hunk 2]
```diff
@@ -242,10 +242,12 @@ lvwClients->setMinimumHeight ( 140 );
     // central server address type combo box
     cbxCentServAddrType->clear();
     cbxCentServAddrType->addItem ( csCentServAddrTypeToString ( AT_DEFAULT ) );
```
Comment (ann0see) [inline src/serverdlg.cpp:244]: Same here

**Comment by corrados** [comment]
I hate compiler macros. Let's keep the code as is right now and the next time we add new lists, we'll change the code to be a loop.

**Comment by ann0see** [comment]
Ok. Sounds good

**Comment by corrados** [comment]
Ok, so let's merge it. Thanks.

File: ChangeLog [hunk 3]
```diff
@@ -2,7 +2,9 @@
 
 
 
-### 3.6.2git <- NOTE: the release version number will be 3.6.3 ###
+### 3.6.2git <- NOTE: the release version number will be 3.7.0 ###
+
+- introduced new server lists: Any Genre 2, Choral/Barbershop (#875)
```
Comment (pljones) [inline ChangeLog:7]: This comment is wrong, by the way.  What the change does is https://github.com/jamulussoftware/jamulus/pull/885/files#diff-40ab92941994ec9418dbb76ae46d5aef0e4cb8b034f2be2b31562bb9aca8070f
- rename "Default" to "Any Genre 1"; rename "All Genres" to "Any Genre 2"; rename "Genre Classical/Folk/Choral" to "Genre Classical/Folk"
- add "Any Genre 3" and "Genre Choral/Barbershop"

**Pull request merged**